src/Sulu/Bundle/TagBundle/Domain/Event/TagRemovedEvent.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TagBundle\Domain\Event;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+
+class TagRemovedEvent extends DomainEvent
+{
+    /**
+     * @var int
+     */
+    private $tagId;
+
+    /**
+     * @var int
+     */
+    private $tagName;
+
+    /**
+     * @var bool
+     */
+    private $tagWasMerged;
+
+    public function __construct(
+        int $tagId,
+        string $tagName,
+        bool $tagWasMerged = false
```

Comment:
should we go here with `array $context = []` to be more flexible and so we can also add e.g.:

```
new TagRemovedEvent(
      $tag->getId(),
      $tag->getName(),
      [
            'merged' => true,
            'mergedTagId' => $targetTag->getId(),
      ]
);
```

Reply:
is okay for me 🙂